src/js_realm.hpp

```diff
@@ -32,6 +32,7 @@
 #include "realm/binary_data.hpp"
 #include "realm/util/functional.hpp"
 #include <exception>
+#include <iostream>
```

Comment:
Why was `iostream` included in this change?

Reply:
Left-over from debugging - I'll remove it